src/app/spec/component-fixture-spec.helper.ts

```diff
@@ -1,13 +1,40 @@
 import {ComponentFixture, TestBed} from '@angular/core/testing';
-import {ChangeDetectorRef, DebugElement, ElementRef, SchemaMetadata, Type} from '@angular/core';
+import {
+    ChangeDetectorRef, DebugElement, SchemaMetadata, Type, Component, ViewChild,
+    ViewContainerRef, ComponentFactoryResolver, Optional, NgModule
+} from '@angular/core';
 import {SelectSpecHelper} from './select-spec.helper';
-import {By} from '@angular/platform-browser';
 import {Predicate} from '@angular/core/src/debug/debug_node';
+import {BrowserDynamicTestingModule} from '@angular/platform-browser-dynamic/testing';
+
+@Component({
+    selector: 'wave-test-host',
+    template: `<ng-template #component></ng-template>`
+})
+class TestHostComponent {
```

Comment:
The IDE shows: "TestHostComponent is not declared in any angular module" 
Is it possible to get rid of this error?

Reply:
I got rid of it by defining a Module and declare it in it..
The reason for that error is that i only declare the Component in the TestBed. The TestBed gives an option to import pre-defined modules though. I just added the TestHostComponent to the EntryModule that declares the generic type for the used TestBed.